### PR TITLE
Finally having sugar on my pizza.

### PR DESCRIPTION
**Pull request by Obyka**
I've implemented my pizza with the tests.

### CODE REVIEW

**Comment by Obyka** [comment]
Closes #287

**Comment by Obyka** [comment]
Good morning,
I'm seeing there is an issue in my PR. 

As you can see, it went all well locally. (Please have a look at this maven screenshot.)
image

If you want, I can correct my commit to have a valid PR. 
Thanks in advance
Florian POLIER

File: src/main/java/ch/heigvd/res/pizza/domain/Obyka/Sugar.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ch.heigvd.res.pizza.domain.wasadigi;
```
Comment (wasadigi) [inline src/main/java/ch/heigvd/res/pizza/domain/Obyka/Sugar.java:1]: This is the wrong package (should be domain.Obyka)

File: src/main/java/ch/heigvd/res/pizza/domain/Obyka/Sugar.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import ch.heigvd.res.pizza.domain.IProduct;
+
+import java.math.BigDecimal;
+
+public class Margherita implements IProduct {
```
Comment (wasadigi) [inline src/main/java/ch/heigvd/res/pizza/domain/Obyka/Sugar.java:7]: This is the wrong class name (should be Sugar)

File: src/test/java/ch/heigvd/res/pizza/domain/Obyka/SugarTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package ch.heigvd.res.pizza.domain.wasadigi;
```
Comment (wasadigi) [inline src/test/java/ch/heigvd/res/pizza/domain/Obyka/SugarTest.java:1]: wrong package

File: src/test/java/ch/heigvd/res/pizza/domain/Obyka/SugarTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  void thePriceAndNameForMargheritaShouldBeCorrect() {
+    Margherita beer = new Margherita();
+    assertEquals(beer.getName(), Margherita.NAME);
+    assertEquals(beer.getPrice(), Margherita.PRICE);
+  }
+
+  @Test
+  void aPizzaioloShouldAcceptAnOrderForMargherita() {
+    Pizzaiolo mario = new Pizzaiolo();
+    String productName = "ch.heigvd.res.pizza.domain.wasadigi.Margherita";
```
Comment (wasadigi) [inline src/test/java/ch/heigvd/res/pizza/domain/Obyka/SugarTest.java:24]: You want to test Sugar

**Pull request closed without merging**